src/generator/AutoRest.Go/Model/MethodGo.cs

```diff
@@ -102,6 +103,19 @@ public string MethodParametersSignature
       }
     }
 
+    public string MethodParametersSignatureLazy
+    {
+      get
+      {     
+        var signature = new StringBuilder("(");
+        signature.Append(MethodParametersSignature);
+        signature.Append(MethodParametersSignature.Length > 0
+          ? ", stop <-chan struct{})"
+          : "stop <-chan struct{})");
```

Comment:
I don't want it to collide with the cancel channel used in long running operations.